mountinfo/mountinfo.go

```diff
@@ -29,35 +29,38 @@ type Info struct {
 	// ID is a unique identifier of the mount (may be reused after umount).
 	ID int
 
-	// Parent indicates the ID of the mount parent (or of self for the top of the
-	// mount tree).
+	// Parent is the ID of the parent mount (or of self for the root
+	// of this mount namespace's mount tree).
 	Parent int
 
-	// Major indicates one half of the device ID which identifies the device class.
-	Major int
+	// Major and Minor are the major and the minor components of the Dev
+	// field of unix.Stat_t structure returned by unix.*Stat calls for
+	// files on this filesystem.
+	Major, Minor int
```

Comment:
Curious; how does the documentation show these combined comments? Does it repeat the same doc for each of them, or does it also should them grouped?

Reply:
1. `go doc` shows it as is:
```
	// Major and Minor are the major and the minor components of the Dev
	// field of unix.Stat_t structure returned by unix.*Stat calls for
	// files on this filesystem.
	Major, Minor int
```

2. `go doc` (a webserver) does exactly the same since it's a struct members:

```
type Info
Info reveals information about a particular mounted filesystem. This struct is populated from the content in the /proc/<pid>/mountinfo file.

type Info struct {
    // ID is a unique identifier of the mount (may be reused after umount).
    ID int

    // Parent is the ID of the parent mount (or of self for the root
    // of this mount namespace's mount tree).
    Parent int

    // Major and Minor are the major and the minor components of the Dev
    // field of unix.Stat_t structure returned by unix.*Stat calls for
    // files on this filesystem.
    Major, Minor int
...
```

Reply:
Ah, yes, had wires crossed and thought we were in a `var()` block 🤦 

makes sense